tests/integration/test_integration_custom_tenant.py

```diff
@@ -0,0 +1,13 @@
+from .asserts import assert_resource
+
+
+def test_3scale_master_url_is_set(master_api, master_url, master_token):
+    assert master_url is not None
```

Comment:
I believe check like this is not needed and `assert master_url` is enough. This is valid comment also for next asserts.